cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -17,6 +17,120 @@
 
     boolean cleanup();
 
+    String DEFAULT_DOCKER_CMEK = "/secrets/storage-cmek.json";
+
+    boolean upload();
```

Comment:
Why are these all "new lines"?

Reply:
Ah. I moved the builder method below the constants and interface methods.